doc/source/examples/multiphysics/rising-bubble-VOF/rising-bubble-VOF.rst

```diff
@@ -0,0 +1,301 @@
+
+We start off with a rectangular mesh that spans the domain defined by the corner points situated at the origin and at point
+:math:`[1,2]`. The first :math:`1,2` couple defines that number of initial grid subdivisions along the length and height of the rectangle. 
+This makes it so our initial mesh is composed of perfect squares. We proceed then to redefine the mesh globally four times by setting
+``set initial refinement=8``. 
+
+.. code-block:: text
+        
+    #---------------------------------------------------
+    # Mesh
+    #---------------------------------------------------
+    subsection mesh
+            set type = dealii
+            set grid type = subdivided_hyper_rectangle
+            set grid arguments = 1, 2 : 0, 0 : 1, 2 : true
+            set initial refinement = 8
+    end
+    
+In the ``mesh adaptation subsection``, adaptive mesh refinement is 
+defined for ``phase``. ``min refinement level`` and ``max refinement level`` are 6 and 8, respectively.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh Adaptation
+    #---------------------------------------------------
+    subsection mesh adaptation
+        set type                    = kelly
+        set variable                = phase
+        set fraction type           = fraction
+        set max refinement level    = 8
+        set min refinement level    = 6
+        set frequency               = 1
+        set fraction refinement     = 0.97
+        set fraction coarsening     = 0.1
+    end
+
+
+
+Call the gls_navier_stokes_2d by invoking:  
+
+``mpirun -np 8 gls_navier_stokes_2d rising_bubble_VOF.prm``
+
+to run the simulation using eight CPU cores. Feel free to use more.
+
+
+.. warning:: 
+    Make sure to compile lethe in `Release` mode and 
+    run in parallel using mpirun 
+
+
+
+
+-------
+Results
+-------
+
+The following image shows the shape and dimensions of the bubble after 3 seconds of simulation, and compares it with results of [1, 2].
+
+.. image:: images/bubble.png
+    :alt: bubble
+    :align: center
+    :width: 400
+
+A python post-processing code `(rising_bubble.py)` 
+is added to the example folder to post-process the results.
+Run ``python3 ./rising_bubble.py ./Output`` to execute this 
+post-processing code, where ``./Output`` is the directory that 
+contains the simulation results. In post-processing, the maximum and minimum axial positions of the light phase (bubble) are tracked to monitor the location of the bubble center as a function of time. Then, the bubble rise velocity is calculated as the derivation of the bubble axial position. These results are compared with the simulations of Zahedi, Kronbichler, and Kreiss [2]. The following images shows the results of these comparisons. The oscillations in the bubble rise velocity are attributed to the different methods used for finding the centroid of the bubble, first order derivation, and smoothing of the bubble location and rise velocity.
```

Comment:
```suggestion
contains the simulation results. In post-processing, the maximum and minimum axial positions of the light phase (bubble) are tracked to monitor the location of the center of the bubble as a function of time. Then, the bubble rise velocity is calculated as the derivation of the bubble axial position. These results are compared with the simulations of Zahedi, Kronbichler, and Kreiss [2]. The following images shows the results of these comparisons. The oscillations in the bubble rise velocity are attributed to the different methods used for finding the centroid of the bubble, first order derivation, and smoothing of the bubble location and rise velocity.
```

Reply:
Might be a good idea to improve this post-processing as a follow up (also maybe using higher-order in time here could help).